cupy/cudnn.py

```diff
@@ -8,10 +8,12 @@
 from cupy.core import internal
 from cupy import cuda
 from cupy.cuda import cudnn
+from cupy.cuda import device
 
 
 _cudnn_version = cudnn.getVersion()
 _thread_local = threading.local()
+_compute_capability = device.Device().compute_capability
```

Comment:
Maybe, we have to call `device.Device()` in `is_tensor_core_available`.
Because, user may use different generation GPU on single machine.

Reply:
All right, will move `device.Device()` to `is_tensor_core_available`.